### PR TITLE
Add guide for fetching value of certain custom metrics

### PR DESCRIPTION
**Pull request by sanwishe**
This pr is used to add guide for fetching certain custom metrics from apiserver.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,10 +65,14 @@ kubectl proxy &
...
 
-```
+```bash
 # you can pipe to `jq .` to pretty-print the output, if it's installed
 # (otherwise, it's not necessary)
 kubectl get --raw "/apis/custom.metrics.k8s.io/v1beta1" | jq .
+# fetching certain custom metrics of namespaced resources
+kubectl get --raw "/apis/custom.metrics.k8s.io/v1beta1/namespaces/default/services/kubernetes/test-metric" | jq .
+# certain custom metrics of  non-namespaced resource
+kubectl get --raw "/apis/custom.metrics.k8s.io/v1beta1/nodes/node1/test-metric" | jq .
```
Comment (yangjunmyfm192085) [inline README.md:75]: In the test example, `curl -XPOST` only sets the value of  `/namespaces/default/services/kubernetes/test-metric`, which `nodes/node1/test-metric` should not be needed here
/cc @dgrisonnet

Comment (sanwishe) [reply README.md]: > In the test example, `curl -XPOST` only sets the value of `/namespaces/default/services/kubernetes/test-metric`, which `nodes/node1/test-metric` should not be needed here /cc @dgrisonnet

done~

**Review by zroubalik — APPROVED** [review]

**Review by yangjunmyfm192085 — COMMENTED** [review]
/ok-to-test

**Pull request merged**